python/pyiceberg/conversions.py

```diff
@@ -109,6 +109,8 @@ def _(primitive_type: BooleanType, value_str: str) -> Union[int, float, str, uui
 @handle_none
 def _(primitive_type: PrimitiveType, value_str: str) -> int:
     """
+    Converts a string to an integer value
```

Comment:
Shouldn't this one be one line up?

Reply:
whoops :) fixed with https://github.com/apache/iceberg/pull/7823/commits/d450891f870e94d59022ca0dbec79e1e667ba4fa